tests/example.c

```diff
@@ -95,27 +95,27 @@ int main()
         printf(" 8. print users\n");
         printf(" 9. count users\n");
         printf("10. quit\n");
-        gets(in);
+        fgets(in, sizeof(in), stdin);
         switch(atoi(in)) {
             case 1:
                 printf("name?\n");
-                add_user(id++, gets(in));
+                add_user(id++, fgets(in, sizeof(in), stdin));
                 break;
             case 2:
                 printf("id?\n");
-                gets(in);
+                fgets(in, sizeof(in), stdin);
                 id = atoi(in);
                 printf("name?\n");
-                add_user(id, gets(in));
+                add_user(id, fgets(in, sizeof(in), stdin));
                 break;
             case 3:
                 printf("id?\n");
-                s = find_user(atoi(gets(in)));
+                s = find_user(atoi(fgets(in, sizeof(in), stdin)));
                 printf("user: %s\n", s ? s->name : "unknown");
                 break;
             case 4:
                 printf("id?\n");
-                s = find_user(atoi(gets(in)));
+                s = find_user(atoi(fgets(in, sizeof(in), stdin)));
```

Comment:
P.S.: Whatever changes we make to `example.c` also need to be reflected in `doc/userguide.txt`.

Reply:
If the patch is finished, there won't be any userguide changes necessary anymore.